conf/upgrade/upgrade_ui.yaml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
You can add some description here in comment and in conf/README.md it should be described there as well as we used to document new config values.